cmd/iso_package/s3.go

```diff
@@ -0,0 +1,106 @@
+package main
+
+import (
+	"crypto/tls"
+	"fmt"
+	"net/http"
+	"net/url"
+	"os"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3/s3manager"
+)
+
+type S3CredentialsConfig struct {
```

Comment:
As commented in the parameters section, you are missing the CA bundle (as a field and in its usage)

Reply:
CA will load in the correct path on the kubernetes build, you do not need to add into a option.

Reply:
I don't "love" it as a solution for a generic command line tool, but since it's internal and we know it will be executed inside a Pod in a Job defined by the controller, I guess it's fine